### PR TITLE
program show range of input number

### PR DESCRIPTION
**Pull request by lukaszkutylowski**

### CODE REVIEW

**Review by MatDls — CHANGES_REQUESTED** [review]
That's great. But let's not stick tho that.
Try to pass argument by args instead of using Scanner.
Can you do that please?:)

**Comment by lukaszkutylowski** [comment]
@MatDls 
args done :)

File: exercisePPJ_07_I/ExercisePPJ_07_I.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+import java.util.Scanner;
+
+public class ExercisePPJ_07_I {
+	public static void main(String[] args) { 
+		//System.out.print("Input value in range 0...10: ");
+		//Scanner input = new Scanner(System.in);
+		//int wrt = input.nextInt();
+		
+		//code change - input from args
```
Comment (MatDls) [inline exercisePPJ_07_I/ExercisePPJ_07_I.java]: Do not leave commented code. (If you ever want to go back to code you once commited - you can do it in git)

File: exercisePPJ_07_I/ExercisePPJ_07_I.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+		
+		String wrtS =  args[0];
+		int wrt = Integer.parseInt(wrtS);
```
Comment (MatDls) [inline exercisePPJ_07_I/ExercisePPJ_07_I.java]: Inline those two lines

File: exercisePPJ_07_I/ExercisePPJ_07_I.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+		if(wrt > 5) {
+			System.out.println("int z przedziału <6, 10>");
+		} else {
+			System.out.println("int z przedziału <0, 5>");
+		}
```
Comment (MatDls) [inline exercisePPJ_07_I/ExercisePPJ_07_I.java]: you have two print statements.
Try do do that with one, and print variable which depends on if condition.
Try to use if only, (without else) ;)

**Review by MatDls — APPROVED** [review]
It's good, but let's practice a little more.
Try to implement my comments before merging:)

**Comment by lukaszkutylowski** [comment]
@MatDls 
all topics done, please check :)

**Pull request merged**